packages/cadl-python/test/generated/lro-basic/azure/lro/_operations/_operations.py

```diff
@@ -148,10 +145,7 @@ def begin_create(self, **kwargs: Any) -> LROPoller[str]:
 
         def get_long_running_output(pipeline_response):
             response = pipeline_response.http_response
-            if response.content:
-                deserialized = response.json()
-            else:
-                deserialized = None
+            deserialized = _deserialize(str, response.json())
```

Comment:
Hi @iscai-msft ,

After this change, currently the mock api testing in pipeline failed because the cadl-ranch mocked response don't has any content.
I think this is a cadl-ranch mockapi problem (because the cadl spec define a string response), so I created a PR to fix it : https://github.com/Azure/cadl-ranch/pull/142.